ui/app/helpers/wizard-constants.js

```diff
@@ -42,6 +42,7 @@ export const DEFAULTS = {
 
 export const FEATURE_MACHINE_STEPS = {
   secrets: {
+    connection: 4, // ARG TODO unsure just making this up for now
```

Comment:
??

Reply:
Yep, need to come back to this. Had a note, and this missed it. Thanks!